tests/dragonfly/cluster_test.py

```diff
@@ -2970,6 +2969,9 @@ async def test_cluster_sharded_pub_sub(df_factory: DflyInstanceFactory):
     consumer.ssubscribe("kostas")
 
     await c_nodes[0].execute_command("SPUBLISH kostas hello")
+    # We need to sleep cause we use DispatchBrief internally. Otherwise we can't really gurantee
```

Comment:
you can not guarantee in any case. how do you know that the message arrive to the client? maybe he is in australia?